app/controllers/v1/supplemental_claims_controller.rb

```diff
@@ -126,6 +131,12 @@ def create_appeal_submission(submitted_appeal_uuid, backup_zip)
       appeal_submission.id
     end
 
+    def handle_saved_claim(form:, guid:, form4142:)
```

Comment:
Could this be a `private` method? Could other methods in this file also be private?

Reply:
We have `private` defined on line 32, so the only public methods are `create` and `show`.

Reply:
oh heh 😅 my bad